spec/lib/twingly/url_spec.rb

```diff
@@ -102,6 +114,16 @@ def valid_urls
       end
     end
 
+    context "IDNA 2003 vs IDNA 2008" do
```

Comment:
BTW, once upon a time I read http://betterspecs.org/ to try to figure out how to use contexts, describes and its.